unit_tests/test_table.py

```diff
@@ -618,9 +620,16 @@ def mock_columns_filter_helper(*args):
         def mock_filter_chain_helper(**kwargs):
             mock_filters.append(kwargs)
             return fake_filter
-
+        
+        row_set = RowSet()
+        row_set.add_row_range(RowRange(row_start, row_stop))
+        
+        def mock_get_row_set_object(*args):
+            return row_set
```

Comment:
Since we are mocking out `_get_row_set_object` here only to test that it is passed to another mock, we don't need to create a real instance:  it can just be an `object()` like `fake_filter`.

Reply:
I did it intentionally, so that we can have the same instance of RowSet (line 650). This is because we don't have __eq__ for row_set. Once its available, we can do as per your suggestion.
I have raised a separate PR for __eq__ for row_set.

Reply:
But we don't *need* an actual `RowSet` instance:  we can just use an `object()` (compared by identity) in the same way as `fake_filter`.

Reply:
yes, got it now. Updated the code. Thanks.